pkg/networkpolicy/controller.go

```diff
@@ -458,8 +458,9 @@ func (c *Controller) Run(ctx context.Context) error {
 			processingTime := float64(time.Since(startTime).Microseconds())
 			packetProcessingHist.WithLabelValues(string(packet.proto), string(packet.family)).Observe(processingTime)
 			packetProcessingSum.Observe(processingTime)
-			packetCounterVec.WithLabelValues(string(packet.proto), string(packet.family)).Inc()
-			klog.V(2).Infof("Finished syncing packet %d took: %v accepted: %v", *a.PacketID, time.Since(startTime), verdict == nfqueue.NfAccept)
+			verdictStr := verdictString(verdict)
+			packetCounterVec.WithLabelValues(string(packet.proto), string(packet.family), verdictStr).Inc()
+			klog.V(2).Infof("Finished syncing packet %d took: %v verdict: %v", *a.PacketID, time.Since(startTime), verdictStr)
```

Comment:
This conflicts with https://github.com/kubernetes-sigs/kube-network-policies/pull/50 so will wait on @uablrek if that's inbound

Reply:
I re-based #50

Reply:
should we close this now that #50 merged?